3.preprocessing_features/2.bulk_processing.ipynb

```diff
@@ -58,6 +58,11 @@
     }
    ],
    "source": [
+    "# Condition for how to normalize the plates\n",
+    "normalize_with_U2OS = (\n",
```

Comment:
Just a general question - why is U20S the sample we're normalizing to?

Reply:
Additionally, is there anything special I should be doing with this sample, either on its own or in relation to others, when I am finding optimal conditions?

Reply:
For context, the U2OS cell line is the cell line that Cell Painting was first optimized to, so we expect to have good results with these cells. This cell line is also consistent across all plates (we can also call it the anchor), so normalizing to this cell line would in theory make it to where we can compare results between plates. But, we don't actually know if that is correct so having both normalizing methods will be good to test while optimizing.

And as far as I know, nothing special should be done with this cell line since I believe we are finding optimal conditions per cell line, so each cell line is treated individually. Feel free to confirm with Greg though!